src/index.js

```diff
@@ -0,0 +1,35 @@
+import './style.css';
+import Task from './modules/tasks.js';
+import displayTask from './modules/displayTask.js';
+
+const newTask = new Task();
```

Comment:
- [ ] According to the project requirements, you are required to have a hardcoded array of task objects with each object having the following properties: description (`string`), completed (`boolean`), index (`number`). Kindly add this array.

Reply:
@TedLivist please I'm not getting a clear picture of this part. how do I fix it

Reply:
@principles31 This Tasks class initializes your tasks array from the localStorage.

Your tasks should be hardcoded at the point not loaded from the localStorage. You can leave your class as it is (since, depending, you may want to use it later) but create an array of tasks, like so:

  ```
  const tasks = [
    {
       description: "First task",
       completed: false,
       index: 1
    },
    {
       description: "Second task",
       completed: true,
       index: 2
    }
  ]
  ```
This `tasks` list is what you should pass to the `displayTask` function as an argument.

Reply:
@TedLivist Thank you for your effort to make my project meet the requirement.